packages/lighthouse-spa/src/app/playground/components/home/home.component.ts

```diff
@@ -211,34 +208,21 @@ export class HomeComponent implements OnInit {
       });
   };
 
-  fetchScore = (auditId): void => {
-    this.appService.fetchScore(auditId).then((responses) => {
-      this.showScore = true;
-      const scores = responses.reduce((acc, val) => {
-        Object.keys(val).map((prop: any) => {
-          if (prop in acc) {
-            acc[prop] += val[prop];
-          } else {
-            acc[prop] = val[prop];
-          }
-        });
-        return acc;
-      }, {});
-
-      this.lhciScores.map((record) => {
-        Object.keys(scores).map((key) => {
-          if (key === record.label) {
-            record.score = Math.trunc((scores[key] / responses.length) * 100);
-            if (record.score >= 0 && record.score <= 49) {
-              record.class = 'orange';
-            } else if (record.score >= 50 && record.score <= 89) {
-              record.class = 'blue';
-            } else if (record.score >= 90 && record.score <= 100) {
-              record.class = 'green';
-            }
-            return record;
+  fetchScore = ( score ) => {
```

Comment:
@riginoommen can you check the code once. I did change here in component part. 

app-outlined-pie-graph just expect score and name of the score. Nothing else. It will internally calculate the color based on that. So i dont think you need to compute the color here

Reply:
Updated the styling section